DP02_Galaxy_Cluster_Weak_Lensing.ipynb

```diff
@@ -0,0 +1,769 @@
+{
```

Comment:
<ul><li>I think it's be good to explain more about the color cuts and why they have those values for each type of galaxy.</li><li>Try to rely not only on color to convey information as suggested in RNT-045, i.e., you could use different symbols.</li><li>Add units to the plot axis (mag for the abscissa).</li><li>Consider using the function to deleting figures after they are not needed, as suggested in RTN-045, for this and te other figures below. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<ol><li>adjusted </li><li>yes, they have different symbols</li><li>added</li><li>I don't think we should delete figures as they are good for visualization and help the user understand the analysis</li></ol>